rootfs/build.sh

```diff
@@ -17,11 +17,15 @@ apk add --update-cache \
   libffi-dev \
   libpq \
   postgresql-dev \
-  python \
-  python-dev
+  python3 \
+  python3-dev \
+  git
+
+# Symlink python3 binary to python
+ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
perhaps it conflicts with the `python` package, so they just want to install it to `/bin/python3`. Either way this works :)